### PR TITLE
Simplify example and test

### PR DESCRIPTION
**Pull request by rob-mccann**
Feel free to review commit by commit

Alternative to #10 which adds docker.

This seems much simpler and easier to test the integration with pdfmake although it removes the integration test with Koa.

### CODE REVIEW

File: example/index.js [hunk 1]
```diff
@@ -1,41 +1,34 @@
-// http app server
-import Koa from 'koa';
+import fs from 'fs';
+import path from 'path';
 
 // library
 import { createElement, createRenderer } from '../dst';
 
 // PDF to render
 import PDF from './components/root';
```
Comment (mtlewis) [inline example/index.js]: Might be worth renaming this to something less generic while we're in here - `InvoicePDF`?

File: test/integration.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import path from 'path';
```
Comment (mtlewis) [inline test/integration.test.js]: cooool

File: test/integration.test.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      const pdfParser = new PDFParser();
+
+      const data = new Promise((resolve, reject) => {
+        pdfParser.on('pdfParser_dataError', errData =>
+          reject(errData.parserError),
+        );
+        pdfParser.on('pdfParser_dataReady', pdfData => resolve(pdfData));
+      });
+
+      pdfParser.loadPDF(path.resolve(__dirname, '../example', 'example.pdf'));
```
Comment (mtlewis) [inline test/integration.test.js]: you could do `path.resolve(__dirname, '../example/example.pdf')` if you wanted it to be slightly more concise ¯\_(ツ)_/¯

File: example/index.js (hunk 1 above)
Comment (rob-mccann) [reply example/index.js]: I thought about that but left it like this to remove extra cognitive load when comprehending the example; I don't feel strongly though.

File: test/integration.test.js (hunk 3 above)
Comment (rob-mccann) [reply test/integration.test.js]: :+1: done - that's the correct way of doing it

File: example/index.js (hunk 1 above)
Comment (mtlewis) [reply example/index.js]: Yep - nbd either way.  Already approved and more than happy for this to be merged as-is :)

**Pull request merged**